### PR TITLE
Ensure calling dispose on a popover will not set the title of the popover next time it is created

### LINKED ISSUE #26847: Calling dispose on a popover may set the title of the popover next time it is created
If the element a popover is attached to has a title, manually calling .popover('dispose') causes the pop over to have a title in the pop over which matches the title attribute of the element.

This can actually replace the title which is specified in the options. See codepen: https://codepen.io/jarstelfox/pen/EpxdrX

BS Version: 4.1.1

### PR DESCRIPTION
**Pull request by jarstelfox**
Fixes #26847

After doing some digging it looks like popover was previously checking for destroy, but this has been renamed to dispose in v4

### CODE REVIEW

File: js/src/popover.js [hunk 1]
```diff
@@ -150,7 +150,7 @@ const Popover = (($) => {
         let data = $(this).data(DATA_KEY)
         const _config = typeof config === 'object' ? config : null
 
-        if (!data && /destroy|hide/.test(config)) {
+        if (!data && /dispose|destroy|hide/.test(config)) {
```
Comment (Johann-S) [inline js/src/popover.js]: Can you remove `destroy` because it doesn't exist anymore

**Review by Johann-S — CHANGES_REQUESTED** [review]

**Comment by jarstelfox** [comment]
I've addressed your requested changes

**Comment by vsn4ik** [comment]
@Johann-S Review please!

**Review by Johann-S — APPROVED** [review]

**Comment by bensamin** [comment]
good

**Pull request merged**